DASH-IF-IOP.bs.md

```diff
@@ -10,9 +10,1070 @@ Live discussion in #document-authoring on Slack.
+
+## Representations ## {#timing-representation}
+
+A <dfn>representation</dfn> is a sequence of <dfn>segment references</dfn> and a description of the samples within the referenced [=media segments=]. Each representation belongs to exactly one adaptation set and to exactly one [=period=], although [[#timing-connectivity|a representation may be connected with a representation in another period]].
+
+Each segment reference addresses a [=media segment=] that corresponds to a specific time span on the [=sample timeline=]. Each [=media segment=] contains samples for a specific time span on the [=sample timeline=].
+
+Note: [=Simple addressing=] allows the actual time span of samples within a [=media segment=] to deviate from the nominal time span described in the MPD. All timing-related clauses in this document refer to the nominal timing described in the MPD unless otherwise noted.
+
+The exact mechanism used to define segment references depends on the [=addressing mode=] used by the representation. All representations in the same adaptation set SHALL use the same [=addressing mode=].
+
+In a static MPD, a representation SHALL contain enough [=segment references=] to cover the entire time span of the [=period=].
+
+<figure>
+	<img src="Images/Timing/StaticMpdMustBeCovered.png" />
+	<figcaption>In a static MPD, the entire period must be covered with [=media segments=].</figcaption>
+</figure>
+
+In a dynamic MPD, a representation SHALL contain enough segment references to cover the time span of the [=period=] that intersects with the [=time shift window=].
+
+Note: [=Media segments=] only become [=available=] when their end point is within the [=availability window=]. It is a valid situation that a [=media segment=] is required to be referenced but is not yet [=available=].
+
+<figure>
+	<img src="Images/Timing/MandatorySegmentReferencesInDynamicMpd.png" />
+	<figcaption>In a dynamic MPD, the [=time shift window=] determines the set of required segment references in each representation. [=Media segments=] filled with gray need not be referenced due to falling outside the [=time shift window=], despite falling within the bounds of a [=period=].</figcaption>
+</figure>
+
+Advisement: A dynamic MPD must remain valid for its entire validity duration after publishing. In other words, a dynamic MPD SHALL supply enough segment references to allow the [=time shift window=] to extend to `now + MPD@minimumUpdatePeriod`, where `now` is the current time according to [[#timing-sync|the synchronized clock]].
+
+An unnecessary segment reference is one that is not defined as required by this chapter.
+
+In a static MPD, a representation SHALL NOT contain unnecessary segment references, except when using [=indexed addressing=] in which case such segment references MAY be present.
```

Comment:
From @haudiobe: What are unnecessary segment references? In MPEG-DASH this is called available and valid.

Reply:
The sentence above defines this term.